plugins/CircuitViewer/io/CircuitLoader.cpp

```diff
@@ -261,23 +299,42 @@ class Impl
                                   {"mesh-folder", _properties.meshFolder}};
         auto model = _scene.createModel();
 
-        // Open Circuit and select GIDs according to specified target
-        const brion::BlueConfig bc(source);
-        const brain::Circuit circuit(bc);
+        // Open circuit and simulation if possible
+        brion::URI uri(source);
+        std::unique_ptr<brain::Simulation> simulation;
+        try
+        {
+            simulation = std::make_unique<brain::Simulation>(uri);
+        }
+        catch (std::runtime_error& error)
+        {
+            BRAYNS_INFO << "Cannot open " + source + " as simulation config: "
```

Comment:
This message is printed always or only when report is requested to load?

Reply:
It's unrelated to report and it will never be printed with BlueConfigs. This code deals with the fact that in SONATA circuit and simulation configs are different things (hooray!) and we want to support both.